packages/editor-ui/src/composables/usePushConnection.test.ts

```diff
@@ -221,5 +221,27 @@ describe('usePushConnection()', () => {
 				expect(spy).toHaveBeenCalledWith(executionId);
 			});
 		});
+
+		describe('nodeExecuteAfter', async () => {
+			it("enqueues messages if we don't have the active execution id yet", async () => {
+				uiStore.isActionActive.workflowRunning = true;
+				const queuePushMessageSpy = vi
+					.spyOn(pushConnection, 'queuePushMessage')
+					.mockImplementation(() => {});
+				const event: PushMessage = {
+					type: 'nodeExecuteAfter',
+					data: {
+						executionId: '1',
+						nodeName: 'foo',
+						data: {} as ITaskData,
+					},
+				};
+
+				const result = await pushConnection.pushMessageReceived(event);
+
+				expect(queuePushMessageSpy).toHaveBeenCalledWith(event, 5);
+				expect(result).toBe(false);
+			});
+		});
```

Comment:
I prefer to avoid mocking/spying on internal methods for unit testing. Could we update these tests to assert that 
1. `pushMessageQueue` has a new entry matching this data
2. `retryTimeout` gets set

Since we already use `vi.useFakeTimers` in this file, we don't need to worry about `setTimeout` triggering.

```suggestion
		describe('nodeExecuteAfter', async () => {
			it("enqueues messages if we don't have the active execution id yet", async () => {
				uiStore.isActionActive.workflowRunning = true;
				const event: PushMessage = {
					type: 'nodeExecuteAfter',
					data: {
						executionId: '1',
						nodeName: 'foo',
						data: {} as ITaskData,
					},
				};

				expect(pushConnection.retryTimeout.value).toBeNull();
				expect(pushConnection.pushMessageQueue.value.length).toBe(0);

				const result = await pushConnection.pushMessageReceived(event);

				expect(result).toBe(false);
				expect(pushConnection.pushMessageQueue.value).toEqual([{ message: event, retriesLeft: 5 }]);
				expect(pushConnection.retryTimeout.value).not.toBeNull();
			});
		});
```